### PR TITLE
Backbone: enable some linters

### PR DESCRIPTION
**Pull request by khasanovbi**
subj

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/backbone/index.d.ts [hunk 1]
```diff
@@ -121,8 +121,8 @@ declare namespace Backbone {
         [event: string]: EventHandler;
     }
 
-    export const Events: Events;
-    interface Events extends EventsMixin {}
+    const Events: Events;
+    type Events = EventsMixin;
```
Comment (jgonggrijp) [inline types/backbone/index.d.ts:125]: This bit I'm a bit unsure about. Would it be possible to leave it out?

With hindsight, `EventsMixin` was a mistake; I wasn't aware at the time that declaration merging was a thing. I've been planning for a while to go back and do the right thing.

**Review by jgonggrijp — COMMENTED** [review]
Looks good, just one concern.

**Review by jgonggrijp — APPROVED** [review]
Thanks!

**Comment by khasanovbi** [comment]
Ready to merge

**Pull request merged**